sites/dotnet-guide/contents/tutorials/resharper-to-rider/getting-started-for-resharper-users/index.md

```diff
@@ -0,0 +1,70 @@
+---
+type: TutorialStep
+date: 2021-01-29
+title: Getting Started
+technologies: [.net,csharp]
+products: [rider,resharper]
+topics: [ide]
+author: khalidabuhakmeh
+subtitle: Learn to create and run your first solution in Rider
+thumbnail: ./thumbnail.png
+---
+
+In this first step on your journey, migrating from Visual Studio + ReSharper to Rider, we'll walk through creating a new Console Application solution and running it in Rider. We'll explain elements along the way and compare them to what you may be familiar with as a Visual Studio user. We believe Visual Studio + ReSharper is an excellent development environment. This guide is for folks looking to explore the possibility of adopting Rider as their primary development environment.
+
+All developers can benefit from reading this quick start guide. Those who may have multiple environments across multiple operating systems will benefit the most, moving from Visual Studio + Resharper to JetBrains Rider. This guide will help developers get up and running; for more in-depth details, we suggest going to [the official JetBrains documentation](https://www.jetbrains.com/help/rider/Introduction.html) after reading this tutorial.
+
+Let's get started!
+
```

Comment:
Maybe insert magic here

Reply:
??? Magic?

Reply:
This reads weird, let me take another stab at it.

Reply:
Regarding "magic", see discussion on YouTrack (tl;dr: ignore for now)